demos/gpt2_text_prediction_demo/python/gpt2_text_prediction_demo.py

```diff
@@ -0,0 +1,184 @@
+
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
+    args.add_argument("-m", "--model", help="Required. Path to an .xml file with a trained model",
+                      required=True, type=Path)
+    args.add_argument("-v", "--vocab", help="Required. Path to the vocabulary file with tokens",
+                      required=True, type=str)
+    args.add_argument("--merges", help="Required. Path to the merges file",
+                      required=True, type=str)
+    args.add_argument("-i", "--input", help="Optional. Input prompt", required=False, type=str, action='append')
+    args.add_argument("--max_sample_token_num", help="Optional. Maximum number of tokens in generated sample",
+                      default=40, required=False, type=int)
+    args.add_argument("--top_k", help="Optional. Number of tokens with the highest probability"
+                                      "which will be kept for generation",
+                      default=0, required=False, type=int)
+    args.add_argument("--top_p", help="Optional. Maximum probability, tokens with such a probability"
+                                      "and lower will be kept for generation",
+                      default=0.9, required=False, type=float)
+    args.add_argument("-d", "--device",
+                      help="Optional. Target device to perform inference on."
+                           "Default value is CPU",
+                      default="CPU", type=str)
+    return parser
+
+
+def main():
+    args = build_argparser().parse_args()
+
+    # load vocabulary file for model
+    vocab = load_vocab_file(args.vocab)
+    log.debug("Loaded vocab file from {}, get {} tokens".format(args.vocab, len(vocab)))
+
+    # create tokenizer
+    tokenizer = Tokenizer(BPE(str(args.vocab), str(args.merges)))
+    tokenizer.pre_tokenizer = pre_tokenizers.ByteLevel(add_prefix_space=False)
+    tokenizer.decoder = decoders.ByteLevel()
+
+    log.info('OpenVINO Inference Engine')
+    log.info('\tbuild: {}'.format(get_version()))
+    ie = IECore()
+
+    # read IR
+    model_xml = args.model
+    model_bin = model_xml.with_suffix(".bin")
+    log.info('Reading model {}'.format(args.model))
+    ie_encoder = ie.read_network(model=model_xml, weights=model_bin)
+
+    # check input and output names
+    if len(ie_encoder.input_info) != 1:
+        raise RuntimeError('The demo expects model with single input, while provided {}'.format(
+            len(ie_encoder.input_info)))
+    if len(ie_encoder.outputs) != 1:
+        raise RuntimeError('The demo expects model with single output, while provided {}'.format(
+            len(ie_encoder.outputs)))
+    input_names = list(ie_encoder.input_info)
+    output_names = list(ie_encoder.outputs)
```

Comment:
Since you require the 1/1 I/O model, you can use string instead of list, e.g.:
```
    input_name = list(ie_encoder.input_info)[0]
    output_name = list(ie_encoder.outputs)[0]
```
here and below

Reply:
Done

Reply:
then better
```python
input_name = next(iter(ie_encoder.input_info))
output_name = next(iter(ie_encoder.outputs))
````

Reply:
Done